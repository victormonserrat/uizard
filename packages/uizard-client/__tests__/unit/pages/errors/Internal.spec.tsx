import InternalError from '~/pages/errors/Internal'
import { render } from '~/test/testing-library'

describe('InternalError', () => {
  it('renders correctly', () => {
    const { container } = render(<InternalError />)

    expect(container).toMatchSnapshot('Default')
  })
})
