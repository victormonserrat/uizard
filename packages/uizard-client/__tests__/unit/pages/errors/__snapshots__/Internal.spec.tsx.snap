// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InternalError renders correctly: Default 1`] = `
<div>
  <p>
    Internal error
  </p>
</div>
`;
